server/src/main/scala/controllers/Application.scala

```diff
@@ -278,14 +340,20 @@ class Application @Inject()(
 
     def actorSystem: ActorSystem = system
 
-    def getCrunchState(pointIntTime: MillisSinceEpoch): Future[Option[CrunchState]] = {
-      if (pointIntTime > 0) {
-        crunchStateAtPointInTime(pointIntTime)
+    def getCrunchStateForDay(day: MillisSinceEpoch): Future[Option[CrunchState]] = {
+      if (isHistoricDate(day)) {
+        crunchStateAtPointInTime(day)
```

Comment:
this should be crunchStateForEndOfDay I think

Reply:
agreed

Reply:
actually, further down we already have a getCrunchStateForEndOfDay. maybe this is correct - let's check

Reply:
yeah, sorry, I meant we should be calling that function here.